src/sonic-config-engine/portconfig.py

```diff
@@ -417,7 +417,7 @@ def parse_platform_json_file(hwsku_json_file, platform_json_file):
 
     for intf in port_dict[INTF_KEY]:
         if intf not in hwsku_dict[INTF_KEY]:
-            raise Exception("{} is not available in hwsku_dict".format(intf))
+            continue
```

Comment:
hi Chris, do you mind to help fix the platform.json instead of hack it here? If the interface doesn't exist, it is reasonable to throw, because things doesn't look right.

Reply:
Hi @r12f, this change is related to the platform.json complications with unused interfaces, from this discussion here: https://github.com/sonic-net/sonic-buildimage/pull/20107#discussion_r1746137197

The platform.json file is common to all hwskus under the same platform folder, so updating platform.json to work with C256S2 would for example break other hwskus such as O128S2.

Atlernatively, if we add the unused interfaces back to the hwsku then we would not need to touch platform.json, but then the unused interfaces would show up in the output of `show interface status`.

Reply:
I see. how about reverse the logic? In my understanding, platform.json is more complete and more recent design than port_config.ini, so if a port exists in the hwsku, it should exist in the platform.json as long as the platform.json file exists.

what do you think?

Reply:
@r12f that sounds like a reasonable alternative, I will test the change and post the update once verified.

Reply:
hi Chris, how things go with the reversed check? : D

Reply:
Hi Riff, interestingly, reversing the logic causes at least one unit test to fail, meaning there is likely a fix to be had in the unit tests as well. I'll take a closer look to see what can be done there.

Reply:
@r12f @ccroy-arista can we introduce dummy entry in hwsku json file :- 

```
    "interfaces": {
        "Ethernet0": {
            "default_brkout_mode": "8x100G"
        },
        "Ethernet8": {
        },   <<< DUMMY
        "Ethernet16": {
            "default_brkout_mode": "8x100G"
        },
        "Ethernet24": {
        }, <<< DUMMY
        "Ethernet32": {
            "default_brkout_mode": "8x100G"
        },
        "Ethernet40": {
        } <<< DUMMY
        "Ethernet48": {
            "default_brkout_mode": "8x100G"
        },
```